.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,24 +16,19 @@ Related ticket(s):
 - [ ] Waited for all CI jobs to pass before requesting a review.
 - [ ] Added/updated tests and documentation where relevant.
 
-## For Reviewer/Developer
+## Merging a feature branch?
 
-- Before merging PR
+1. Use the `Squash and merge` option to keep the target branch's commit history nice and clean.
+2. Use the message format `DF-XXX: Ticket name / short description` when merging.
 
-    `main` branch :
+## Merging a release branch?
 
-        - check PR merge tile begins with `release/<major.minor.patch>:<default/custom description>`
-        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
-        - select `Create a merge commit`
+1. Use the `Create a merge commit` option to preserve the original commit IDs.
+2. Use the message format `release/<major.minor.patch>` when merging.
 
-    `develop` branch:
+## Deployment guidance
 
-        - check if merges into base branch are `not to be kept On Hold` and then proceed.
-        - check PR merge tile begins with ticket number Ex `DF-XXX: Ticket name / short description`
-        - select `Squash and merge`
-
-- After merging PR:
-
-  - check and arrange to deploy on platform-sh.
-  - update status on JIRA ticket on deployment.
-  - update slack on deployment progress of the JIRA ticket once deployed.
+1. Check on the `#dev-etna` Slack channel whether it's okay to deploy.
+2. To start a deployment, push the relevant branch to the `platform` remote.
+3. Update the `#dev-etna` Slack channel when the deployment is complete.
```

Comment:
Deployment mentions in slack is currently given in `#dev-etna-dev` or `respective team channels` for that PR where it requires attention to most of us as we all are in those.

Reply:
It would be nice if we could make it a bit more consistent. If all devs are in `#dev-etna`, maybe just send there for cross-team visibility without the repetition?